app/channels/api/v1/global_race_channel.rb

```diff
@@ -0,0 +1,55 @@
+class Api::V1::GlobalRaceChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from('global_channel')
+  end
+
+  def unsubscribed
+    stop_all_streams
+  end
+
+  def create_race(data)
+    return if current_user.blank?
+    return if data['race_type'].blank?
+
+    race_type = Raceable.race_from_type(data['race_type'])
+    return if race_type.nil?
+
+    race = race_type.new
+    case race
+    when StandardRace
+      category = Category.find_by(id: data['category_id'])
+
+      race.category = category
+    when BingoRace
+      game = Game.find_by(id: data['game_id'])
+
+      race.game = game
+      race.card = data['bingo_card']
+    when RandomizerRace
+      game = Game.find_by(id: data['game_id'])
+
+      race.game = game
+      race.seed = data['seed']
+    end
+
+    race.owner = current_user
+    race.status_text = Raceable::OPEN_ENTRY
+    race.visibility = data['visibility'] if race.class.visibilities.key?(data['visibility'])
```

Comment:
Since we defined it as an enum, rails throws an error when setting this value to anything not defined in our enum hash.  This was the workaround so that errors aren't being raised since they're quite slow.